.github/workflows/build-test-ship.yaml

```diff
@@ -25,7 +29,7 @@ on:
         required: true
       ECR_SECRET_ACCESS_KEY:
         required: true
-      GHA_DEV_KEY:
+      NPM_WRITE_TOKEN:
```

Comment:
`NPM_WRITE_TOKEN` is a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for the `deployatniche` user, which we need in order to have permission to check out submodules from the `development` repo in the `run-api-proctor-tests` job below. See [this slack thread](https://nicheinc.slack.com/archives/C0FTZM5H7/p1662064211591039).

Reply:
Was `GHA_DEV_KEY` Defunct? I don't see any other changes to it, and it's not in the current version of the file. Not a major factor either way, just curious.

Reply:
In more detail than is necessary... That was a relic from when I first factored out `build-test-ship` into a single reusable workflow. I actually started out with the `feature/api-proctor` branch and then cut out the API Proctor stuff. When I did that, I accidentally left that secret in the requirements, even though we weren't using it yet, and I just didn't bother fixing it because it didn't hurt, and I thought we'd need it later (this PR) anyway, to clone the `development` environment.

But it turned out that `GHA_DEV_KEY` wasn't powerful enough: it doesn't have read access to all the submodules, which we ended up needing, so I swapped it out for `NPM_WRITE_TOKEN`.

**tl;dr: `GHA_DEV_KEY` was never actually used.**